src/Essentials/src/Geolocation/Geolocation.uwp.cs

```diff
@@ -9,6 +9,13 @@ namespace Microsoft.Maui.Devices.Sensors
 {
 	partial class GeolocationImplementation : IGeolocation
 	{
+		Geolocator? listeningGeolocator;
+
+		/// <summary>
+		/// Indicates if currently listening to location updates while the app is in foreground.
+		/// </summary>
+		public bool IsListeningForeground { get => false; }
```

Comment:
Should this always be false here? I see code to do listening on this platform...

Reply:
You're right, I fixed it.